fury/actor.py

```diff
@@ -2383,8 +2383,8 @@ def billboard(centers, colors=(0, 1, 0), scales=1, vs_dec=None, vs_impl=None,
     return bb_actor
 
 
-def vector_text(text='Origin', pos=(0, 0, 0), scale=(0.2, 0.2, 0.2),
-                color=(1, 1, 1)):
+def vector_text(text='Origin', pos=(0, 0, 0), direction=None,
```

Comment:
Please, do not change the order of the argument. Otherwise, we need to start a deprecation cycle.

some people are using this function for a long time and the next release will mess up their code if we keep it like that.

Also, we should keep the default behavior for the same reason.